src/plugins/LADTable/LADTableView.js

```diff
@@ -34,7 +34,7 @@ export default class LADTableView {
     this._destroy = null;
   }
 
-  show(element) {
+  show(element, isEditing, { renderWhenVisible }) {
```

Comment:
If practical, should we make the second parameter an options object? That way it's clearer when its called:

`show(element, { isEditing: false, renderWhenVisible })`

Reply:
this would apply to all the `show()` methods

Reply:
ah shoot. if this is just how that's been defined historically we probably shouldn't change that in this PR

Reply:
Yeah, although I agree it's desirable it would be a breaking change to the view provider API, which is a big deal. Let's leave it for now.